contents/overlay/dialogs/input-dialog/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { TriangleAlert } from "@yamada-ui/lucide"
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogHeader,
+  Input,
+  Text,
+  useDisclosure,
+  useUpdateEffect,
+  VStack,
+} from "@yamada-ui/react"
+import { useState, type FC } from "react"
+
+const keyword = "delete"
+const InputDialog: FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const [monitoringInput, setMonitoringInput] = useState(true)
+  const [text, setText] = useState<string>("")
+  const handleOnClose = () => {
+    onClose()
+    setText("")
+  }
+  useUpdateEffect(() => {
+    if (text === keyword) {
+      setMonitoringInput(false)
+    } else {
+      setMonitoringInput(true)
+    }
+  }, [text])
+
+  return (
+    <>
+      <Button onClick={onOpen}>Open Dialog</Button>
+      <Dialog
+        isOpen={isOpen}
+        onClose={handleOnClose}
+        cancel="Cancel"
+        onCancel={handleOnClose}
+        success={{
+          colorScheme: "red",
```

Comment:
```suggestion
          colorScheme: "danger",
```

I think you should use `danger` rather than `red`.

Reply:
Applied it!

Reply:
Thanks!